infra/ansible/roles/knot-authoritative/templates/knot.conf.j2

```diff
@@ -25,10 +25,23 @@ template:
     global-module: mod-stats
     storage: /var/lib/knot/zones
 
+key:
+  - id: doh.mesh.nycmesh.net
+    algorithm: hmac-sha384
```

Comment:
512?

Reply:
Good call, it supports it. Regenerated and updated the algo